app/templates/events/view/scheduler.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui grid">
   <div class="row">
-    <div class="sixteen wide mobile three wide computer column">
+    <div class="sixteen wide mobile three wide computer column unscheduled-navbar">
```

Comment:
is it related to this issue.

Reply:
No sir, Actually this is part of another issue which I was solving, but when I made a PR for this ,this came along I searched but didn't get a way to undo this, Can you please help me with that.